src/Core/CustomMethods.php

```diff
@@ -171,11 +171,14 @@ protected function getExtraMethodConfig($method)
         }
         // Lazy define methods
         $lowerClass = strtolower(static::class);
-        if (!isset(self::class::$extra_methods[$lowerClass])) {
+        $lowerMethod = strtolower($method);
+        if (!array_key_exists($lowerClass, self::class::$extra_methods)
+            || !array_key_exists($lowerMethod, self::class::$extra_methods[$lowerClass])
+        ) {
```

Comment:
This feels like it's masking a regression rather than really fixing the problem.

This condition is intended to result in calling `defineMethods()` any time it _hasn't been done yet for this class_.

The change will run the `defineMethods()` code _any time a method doesn't exist_, regardless of whether `defineMethods()` has already been run for this class.

Reply:
I'll split this off as a separate PR so the others can be merged